scipy/stats/stats.py

```diff
@@ -8121,6 +8121,281 @@ def brunnermunzel(x, y, alternative="two-sided", distribution="t",
     return BrunnerMunzelResult(wbfn, p)
 
 
+def _confint_lowerbound(n, quantile, confidence):
+    r"""
+    Compute the lower bound for a one-sided confidence interval
+    for a given
+    - quantile (0<`quantile`<1)
+    - confidence level (0<`confidence`<1)
+    - number of samples `n`.
+
+    Returns the largest index of the sample being a valid lower bound,
+    or `None` if there are not enough samples to derive one.
+
+    Used by the public function confint_quantile().
+
+    .. versionadded:: 1.7.0
+    """
+
+    # compute all probabilities from the binomial distribution for
+    # the quantile of interest
+    bd = distributions.binom(n, quantile)
+
+    # the lower bound is the last index before the invert survival
+    # function value for the target confidence level
+    lb = bd.isf(confidence) - 1
```

Comment:
This is the off-by-one possibility I mentioned, and I think it makes sense the way you have it. The intent of subtracting one is that it guarantees that:
```
bd.sf(lb) >= confidence > bd.sf(lb+1)
```
Is that right?

Reply:
yes